style.css

```diff
@@ -43,8 +38,81 @@ footer ul {
   padding-left: 0;
 }
 
-.menu-bar ul {
+.menu-bar {
+  font-family: inter, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  color: #fff5e1;
+  font-size: 36px;
+  padding-right: 24px;
+  margin-top: 19px;
+  margin-bottom: 19px;
+  position: fixed;
+  top: -1.2rem;
+  background-color: #212121;
+  width: 100%;
+  height: 100vh;
+
+  /* display: flex; */
+  flex-direction: column;
+  align-items: center;
+  padding-top: 10rem;
+  transition: all ease-in-out 0.2s;
   display: none;
+  z-index: 1;
+}
+
+.active {
+  display: block;
+  transition: all ease-in-out 0.2s;
+}
+
+.menu-bar ul {
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  gap: 1rem;
+  padding: 0 0 0 6rem;
+}
```

Comment:
- [ ] Nice work making the changes :clap: However your design looks different to the Figma design. Kindly stick to the Figma design as much as possible.
### Your design :point_down: 
![Screenshot from 2023-05-23 10-19-23](https://github.com/ClarenceAtim/Portfolio/assets/91142432/2e292c78-81b6-4ae6-8589-6d9384a6aa8f)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-05-23 08-05-08](https://github.com/ClarenceAtim/Portfolio/assets/91142432/4c908d14-8bac-4bbb-983f-865789c93fcf)

_**HIT** Your can use this code for the `.menu-bar ul`_
```css
.menu-bar ul {
    display: flex;
    flex-direction: column;
    list-style: none;
    gap: 1rem;
    padding: 0;
    margin-top: -87px;
}
```